src/components/pages/Analytics/TechStackGraph.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import embed from 'vega-embed';
+export default async function TechStackGraph() {
+  let url = `${process.env.REACT_APP_DS_API_URL}/graph/tech-stack-by-role`;
+  // embed is an asynchronous api call, resolves in a promise
+  // for more information visit https://github.com/vega/vega-embed
+  useEffect(() => {
+    const getTechStackData = async () => {
+      await embed('#vis', url);
+    };
+    getTechStackData();
+  }, [url]);
+  return (
+    <>
+      <div id="vis"></div>
```

Comment:
Returning a single div like this makes the code file so much cleaner. Great job.